Update Visualizing-ROS-2-Data-With-Foxglove-Studio.rst (#5053)
ros2/ros2_documentation: source/How-To-Guides/Visualizing-ROS-2-Data-With-Foxglove-Studio.rst

Visualizing ROS 2 data with Foxglove Studio
===========================================

> `Foxglove Studio <https://foxglove.dev/studio>`__ is an open closed source visualization and debugging tool for your robotics data.

Reviewer: removing `open source` is correct here, how about the following?

```suggestion
`Foxglove Studio <https://foxglove.dev/studio>`__ is a visualization and observability platform for robotics development.
```

> **NOTE: Foxglove Studio is no longer open-source.  An open-source alternative that also supports the MCAP file format is** `Rerun.io <https://rerun.io>`__ , **with source code available at https://github.com/rerun-io/rerun.**

Reviewer: this is the dedicated documentation how to use foxglobe studio with ROS 2.
IMO if we want to introduce `rerun`, we should have dedicated and whole procedure how to use `rerun` instead.

Reviewer 2: I concur here. While I think it is fine to mention that Foxglove is closed source and that there are open source alternatives it is pretty unsatisfying that there is no documentation to point the end user to. 

The Rerun team is pretty friendly, I'll give them a ping.

Reviewer 2: I heard back from Rerun. According to them, "I unfortunately wouldn't say that we're all the way there to be able to claim that Rerun has out of the box support for mcap files with ROS 2 messages yet."

Given that this is straight from the project maintainer I would be reluctant to point users in the direction of Rerun. @edgimar I appreciate the spirit of this PR, but given this information I recommend that we not merge your pull request. We would be happy to have your help on some other docs issues though!

Author: @kscottz, that makes sense to me.  Do you think the page should be updated so it accurately communicates the open/closed-source status of Foxglove?

Reviewer 2: Personally, I'm ambivalent until we have an alternative.
